sceptre/plan/actions.py

```diff
@@ -82,8 +82,12 @@ def create(self):
             ],
         }
 
-        if self.stack.on_failure:
+        # can specify either DisableRollback or OnFailure , but not both
+        if self.stack.disable_rollback:
+            create_stack_kwargs.update({"DisableRollback": self.stack.disable_rollback})
+        elif self.stack.on_failure:
             create_stack_kwargs.update({"OnFailure": self.stack.on_failure})
```

Comment:
Where are you going to validate that we don't use both?

Reply:
have not done it yet but will add a test.

Reply:
tests have been added